psi4/src/export_dpd.cc

```diff
@@ -0,0 +1,74 @@
+/*
+ * @BEGIN LICENSE
+ *
+ * Psi4: an open-source quantum chemistry software package
+ *
+ * Copyright (c) 2007-2021 The Psi4 Developers.
+ *
+ * The copyrights for code used from other parties are included in
+ * the corresponding files.
+ *
+ * This file is part of Psi4.
+ *
+ * Psi4 is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * Psi4 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with Psi4; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * @END LICENSE
+ */
+
+#include "psi4/pybind11.h"
+
+#include "psi4/libdpd/dpd.h"
+#include "psi4/libmints/matrix.h"
+
+using namespace psi;
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+void export_dpd(py::module &m) {
+    py::class_<dpdbuf4, std::shared_ptr<dpdbuf4>>(m, "dpdbuf4", "docstring")
+        .def("axpy_matrix", &dpdbuf4::axpy_matrix, "docstring")
```

Comment:
These functions are very self-explanatory, but it still wouldn't hurt to fill in the docstrings, if you felt like it